packages/appstore/src/components/add-options/add-options.tsx

```diff
@@ -1,17 +1,17 @@
-import React, { HTMLAttributes } from 'react';
+import React from 'react';
 import { Text } from '@deriv/ui';
 import classNames from 'classnames';
 import { Icon, DesktopWrapper, MobileWrapper } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import { useStores } from 'Stores';
 
-export interface TAddOptionsProps extends HTMLAttributes<HTMLDivElement> {
-    onClickHandler: () => void;
-    class_names?: string;
-    title: string;
-    description: string;
+interface TAddOptionsProps {
+    numberofAccounts: number;
 }
 
-const AddOptions = ({ title, description, onClickHandler, class_names }: TAddOptionsProps) => {
+const AddOptions = ({ numberofAccounts }: TAddOptionsProps) => {
```

Comment:
This component suppose to be a pure component, don't put hard coded title, description, ... in it, try to reuse it with needed props.